src/components/ProgramList.js

```diff
@@ -196,8 +196,9 @@ class ProgramList extends Component {
           color="primary"
           size="large"
           startIcon={<Add />}
-          href="/mission-control"
```

Comment:
Somewhat unrelated to this PR.
Using `href` was doing a hard re-route and clearing our rover connection. This change makes it use react-router and preserves our connection.